presto-docs/src/main/sphinx/functions/map.rst

```diff
@@ -105,20 +105,20 @@ Map Functions
 
 .. function:: map_top_n_keys(x(K,V), n) -> array(K)
 
-    Returns top n keys in the map ``x``.
+    Returns top n keys in the map ``x`` by sorting its keys in descending order.
     ``n`` must be a non-negative integer
```

Comment:
```suggestion
    ``n`` must be a non-negative integer.
    
```
Suggest adding a period at the end of this sentence, and adding a line to put "For bottom ``n`` keys," at the left margin so it better introduces its code example.